README.md

```diff
@@ -1,79 +1,112 @@
 # Digital Paper Edit - Infrastructure
 
-This is purely the infrastructural part to tie together the [API](https://github.com/bbc/digital-paper-edit-api/) and the [client](https://github.com/bbc/digital-paper-edit-client/) of Digital Paper Edit.
+This is purely the infrastructural part to tie together with
+[firebase](https://github.com/bbc/digital-paper-edit-firebase/) of Digital Paper
+Edit.
 
-[See here for overall project architecture info](https://github.com/bbc/digital-paper-edit-client#project-architecture)
+| Environment | URL                                               |
+| :---------- | :------------------------------------------------ |
+| Test        | <https://digital-paper-edit.test.tools.bbc.co.uk> |
+| Live        | <https://digital-paper-edit.live.tools.bbc.co.uk> |
 
-<!-- [Link to API]() -->
-
-| Environment | URL                                             |
-| :---------- | :---------------------------------------------- |
-| Mock        | Local dev - TBC                                 |
-| Int         | https://digital-paper-edit.int.tools.bbc.co.uk  |
-| Test        | https://digital-paper-edit.test.tools.bbc.co.uk |
-| Live        | https://digital-paper-edit.live.tools.bbc.co.uk |
-
-[See @bbc/digital-paper-edit-client/docs/ADR/2019-04-23-transcript-architecture.md ](https://github.com/bbc/digital-paper-edit-client/blob/master/docs/ADR/2019-04-23-transcript-architecture.md) for more info on the architecure and diagram.
+For historical undestanding of this repo, read
+[ADR 2019-04-23-transcript-architecture.md](https://github.com/bbc/digital-paper-edit-client/blob/master/docs/ADR/2019-04-23-transcript-architecture.md)
```

Comment:
[This](https://github.com/bbc/digital-paper-edit-client/blob/2f27a3c5f53ca1260d5209382d36f142fa15ecad/docs/ADR/2019-04-23-transcript-architecture.md) looks to just have disappeared into the aether at some point?